android-benchmark/benchmark/build.gradle

```diff
@@ -37,12 +37,19 @@ android {
   }
 }
 
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
+  kotlinOptions {
+    jvmTarget = "1.8"
+  }
+}
+
 dependencies {
-  implementation "androidx.benchmark:benchmark:1.0.0-alpha01"
-  implementation "androidx.test:runner:1.1.1"
-  implementation "androidx.test:rules:1.1.1"
-  implementation "androidx.test.ext:junit:1.1.0"
-  implementation "org.jetbrains.kotlin:kotlin-stdlib:1.3.31"
+  implementation "androidx.benchmark:benchmark-common:${deps.versions.androidBenchmark}"
```

Comment:
Let's pull these dependencies out?

Reply:
I don't follow, the version is already shared

Reply:
I meant the maven coordinates themselves. Or do you usually wait for the dependency to show up in a second place before extracting them into `dependencies.gradle`?

Reply:
Yeah for gradle plugin dependencies like these that are only defined once, I'm not too worried about moving the whole coordinate into dependencies.gradle